src/jllvm/materialization/ByteCodeCompileLayer.cpp

```diff
@@ -104,6 +104,44 @@ llvm::AttributeList getABIAttributes(llvm::FunctionType* functionType)
     return llvm::AttributeList::get(functionType->getContext(), llvm::AttributeSet{}, retAttrs, paramAttrs);
 }
 
+/// Class for operand stack
+/// This class also offers method to get and set the current top of stack in order to consider the control flow path
+class OperandStack
+{
+    std::vector<llvm::AllocaInst*> m_values;
+    llvm::AllocaInst** m_topOfStack;
+    llvm::IRBuilder<>& m_builder;
+
+public:
+    OperandStack(u_int16_t maxStack, llvm::IRBuilder<>& builder, llvm::LLVMContext& context): m_builder(builder), m_values(std::vector<llvm::AllocaInst*>(maxStack)), m_topOfStack(m_values.data())
```

Comment:
You don't need the `context` parameter as you can just use `builder.getContext()` in the method.